scripts/install-peer-deps.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+const assert = require('assert');
+const { execSync } = require('child_process');
+const blgr = require('blgr');
+
+const dir = __dirname;
+const root = path.dirname(dir);
+
+const pkg = path.resolve(root, 'package.json');
+const modules = path.resolve(root, 'node_modules');
+
+const json = require(pkg);
+
+const { peerDependencies } = json;
+
+const install = [];
+
+// TODO: update to properly capture prefix ~ and ^
+const regex = /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*)?(\+[0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*)?$/;
```

Comment:
is this for validating npm package names? If so there are published packages for validating names and semver that I think would be better to use.

Reply:
I was trying to not install extra packages but that can work

Reply:
Ah gotcha. I think it’s ok to include as dev dependency. Also helps us stay up to date with npm requirements since they maintain it.